unit-tests/live/image-quality/test-image-quality.py

```diff
@@ -0,0 +1,148 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+
+import pyrealsense2 as rs
+from rspy.stopwatch import Stopwatch
+from rspy import test, log
+import time
+import sys
+import numpy as np
+import cv2
+from scipy.ndimage import zoom
+
+
+# Start depth + color streams and go through the frame to make sure it is showing a depth image
+# Color stream is only used to display the way the camera is facing
+# Verify that the frame does indeed have variance - therefore it is showing a depth image
+
+dev = test.find_first_device_or_exit()
+
+cfg = rs.config()
+cfg.enable_stream(rs.stream.depth, rs.format.z16, 30)
+cfg.enable_stream(rs.stream.color, rs.format.bgr8, 30)
+
+
+def display_image(windowTitle, img):
+    """
+    Display a given image and exits when the x button or esc key are pressed
+    """
+    cv2.imshow(windowTitle, img)
+    while cv2.getWindowProperty(windowTitle, cv2.WND_PROP_VISIBLE) > 0:
+        k = cv2.waitKey(33)
+        if k == 27:  # Esc key to stop
+            cv2.destroyAllWindows()
+            break
+        elif k == -1:  # normally -1 returned,so don't print it
+            pass
+
+
+def save_image(depth, color, filename, showImage):
+    colorizer = rs.colorizer()
+    depth_image = np.asanyarray(colorizer.colorize(depth).get_data())
+    color_image = np.asanyarray(color.get_data())
+
+    depth_rows, _, _ = depth_image.shape
+    color_rows, _, _ = color_image.shape
+
+    # resize the image with the higher resolution to look like the smaller one
+    if depth_rows < color_rows:
+        color_image = zoom(color_image, (depth_rows/color_rows, depth_rows/color_rows, 1))
+    elif color_rows < depth_rows:
+        depth_image = zoom(depth_image, (color_rows/depth_rows, color_rows/depth_rows, 1))
+
+    img = np.concatenate((depth_image, color_image), axis=1)
+
+    if filename != "":
+        cv2.imwrite(filename, img)
```

Comment:
Can we write the images with native python API (cv2 should be pip installed IMO)
And if we drop the side by side image we can drop the numpi as well right?

Reply:
Seems like we can't write images with native python as far as I'm aware, since the image view/save is only relevant in manual debug we moved those requirements into the appropriate function.